extension/goplugin/environment.go

```diff
@@ -691,7 +619,7 @@ func (env *Environment) Clone() extension.Environment {
 		databaseImpl: env.databaseImpl.Clone(),
 
 		name:    env.name,
-		traceID: newTraceID(),
+		traceID: fmt.Sprintf("%s-clone", env.traceID),
```

Comment:
If env is cloned multiple times,  multiple `<root_trace_uuid>-clone` envs may be produced so it's hard to distinguish each other. How about `<root_trace_uuid>-clone-<some_rand>`?

Reply:
How about this?

Reply:
sorry, didn't manage to complete it yesterday. 3 random decimal digits added